sentry-native-sample/build.gradle.kts

```diff
@@ -70,7 +70,7 @@ android {
 dependencies {
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
 
-    implementation("io.sentry:sentry-android:2.0.0-beta02")
+    implementation("io.sentry:sentry-android:2.0.0-rc01")
```

Comment:
Is this supposed to go in this PR?

Reply:
it could be separated, but it doesn't affect anything, my bad